lib/hammer_cli_foreman/combinations.rb

```diff
@@ -0,0 +1,68 @@
+module HammerCLIForeman
+  class Combinations < HammerCLIForeman::Command
+    resource :template_combinations
+    command_name 'combinations'
+    desc _("Manage template combinations")
+
+    module RequestParams
+      def request_params
+        combination_params = { 'hostgroup_id' => params['hostgroup_id'].to_s,
+                               'environment_id' => params['environment_id'].to_s }
+        super.merge('template_combination' => combination_params)
+      end
+    end
+
+    class InfoCombination < HammerCLIForeman::InfoCommand
+      output do
+        field :id, _('ID')
+        field :provisioning_template_id, _('Provisioning template ID')
+        field :provisioning_template_name, _('Provisioning template name')
+        field :hostgroup_id, _('Hostgroup ID')
+        field :hostgroup_name, _('Hostgroup name')
+        field :environment_id, _('Environment ID')
+        field :environment_name, _('Environment name')
+        field :config_template_id, _('Config template ID')
+        field :config_template_name, _('Config template name')
+
+        HammerCLIForeman::References.taxonomies(self)
+        HammerCLIForeman::References.timestamps(self)
+      end
+
+      build_options
+    end
```

Comment:
There should probably be one empty line between the classes

Reply:
done :) thanks!